examples/deps/repositories.bzl

```diff
@@ -225,6 +225,7 @@ def repositories():
         name = "apr_util",
         build_file_content = all_content,
         strip_prefix = "apr-util-1.6.1",
+        sha256 = "b65e40713da57d004123b6319828be7f1273fbc6490e145874ee1177e112c459",
```

Comment:
Just an observation - I'm not sure that Buildifier is checking the `examples` WORKSPACE. The ordering of where `sha256` is in this file is inconsistent which suggests Buildifier isn't running on this file.

Reply:
I've made a TODO for myself to fix this. But I'd like to do it in a separate ticket. Forgot I even included this change in this PR.

Reply:
TODO is here: https://github.com/bazelbuild/rules_foreign_cc/pull/485#issuecomment-779887617